### PR TITLE
Warn user on column error

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?166341#926069

selection_027

enchantress @snopoke

### CODE REVIEW

File: corehq/apps/importer/templates/importer/excel_fields.html [hunk 1]
```diff
@@ -107,6 +107,44 @@
             });
 
             $('#autofill').click(autofillProperties);
+
+            function validateSubmitButton() {
+                var invalidFields = $('.excel_field option:selected')
+                                      .filter(function(k, v) { 
+                                          return /\s+/.test(v.value.trim());
+                                      }).length;
+                if (invalidFields > 0) {
```
Comment (benrudolph) [inline corehq/apps/importer/templates/importer/excel_fields.html]: could just be: `$('.form-actions [type=submit]').prop('disabled', invalidFields > 0)`

File: corehq/apps/importer/templates/importer/excel_fields.html [hunk 2]
```diff
@@ -107,6 +107,44 @@
...
+            function populateRowErrors(tr) {
+                var field = tr.find('.excel_field'), selected;
+
+                if (field.length === 0) { return; }
+
+                selected = field.val().trim();
+                if (/\s+/.test(selected)) {
+                    tr.addClass('error');
+                    tr.find('.errorMsg')
+                      .text('{% trans "This column contains a whitespace character" %}')
```
Comment (benrudolph) [inline corehq/apps/importer/templates/importer/excel_fields.html]: i wonder if this gets picked up as needed to be translated since it's in a JS file

**Comment by benrudolph** [comment]
None of my comments are blocking, i am a little surprised that the template engine was able to parse the `{% trans` tag after the template was rendered

File: corehq/apps/importer/templates/importer/excel_fields.html (hunk 2 above)
Comment (emord) [inline corehq/apps/importer/templates/importer/excel_fields.html]: Yeah. i ran makemessages and it got picked up

File: corehq/apps/importer/templates/importer/excel_fields.html (hunk 1 above)
Comment (emord) [inline corehq/apps/importer/templates/importer/excel_fields.html]: Much nicer.

File: corehq/apps/importer/templates/importer/excel_fields.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/importer/templates/importer/excel_fields.html]: :+1:

**Comment by benrudolph** [comment]
no tests use this code, merging

**Pull request merged**